learn/educators/python.md

```diff
@@ -0,0 +1,98 @@
+---
+Order: 4
+Area: educators
+TOCTitle: Python
+ContentId: e58523d5-e243-45c2-8306-7114077d81ae
+PageTitle: Python in Visual Studio Code
+DateApproved: 02/17/2021
+MetaDescription: Python in Visual Studio Code
+---
+
+# Python
+
+Visual Studio Code is free professional editor that fully supports Python and other features like real-time collaboration. It's highly customizable to support your classroom the way you like to teach.
```

Comment:
"Visual Studio Code is a free, professional-grade, source code editor that fully supports Python and useful features such as real-time collaboration. It's highly customizable to support your classroom the way you like to teach."